src/components/elements/upload/fileDialog/FilePreviewDialog.tsx

```diff
@@ -1,14 +1,11 @@
 import { DialogProps, Paper, Stack } from '@mui/material';
-import pdfWorker from 'pdfjs-dist/build/pdf.worker.js?url';
```

Comment:
it makes more sense for this to live in FilePreview since that component is actually rendering the pdf previews